portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Comments {
+
+  private List<String> name = new ArrayList<>();
+  private List<String> text = new ArrayList<>();
+  private List<Boolean> like = new ArrayList<>();
+
+  public void addName(String inputName){
```

Comment:
Follow with the previous comment, we can define an addComment method. Its signature can be either of

public void addComment(Comment comment)
public void addComment(String name, String text, Boolean like)

The difference between the code is whether you want to expose "Comment" class to the code which executes the addComment method. In our case, both are OK.

Reply:
Originally, I was doing it the second way but I thought that it would be more transparent to add individual methods. Following the "Comment" object structure though this makes more sense.